nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/pom.xml

```diff
@@ -56,6 +56,11 @@ language governing permissions and limitations under the License. -->
             <version>${es.version}</version>
         </dependency>
         <dependency>
+            <groupId>com.squareup.okhttp</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>2.7.1</version>
```

Comment:
That's fair but in response to an issue found by a user on the mail list[1] invokeHttp will need to be updated anyway. So might as well code this on OkHttp v3

[1] https://issues.apache.org/jira/browse/NIFI-2162